starter-code/scripts/articleView.js

```diff
@@ -37,11 +37,15 @@ articleView.handleAuthorFilter = function() {
     //         defining. "$(this)" is using jQuery to select that element, so we can chain jQuery methods
     //         onto it.
     if ($(this).val()) {
+      $('.article-body').hide();
```

Comment:
when filtering, the entire `<article>` should be hidden, not just the `.article-body`
should be `$('article').hide();`